src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -200,7 +211,7 @@ protected void EnableForeignKeyConstraints(Connection conn) {
 
       conn.commit();
     } catch (SQLException se) {
-      LOG.debug(se.getMessage());
+      LOG.error("Could not create foreign keys" + se);
```

Comment:
nit - should we stick with se.getMessage() instead of se.toString()?

Reply:
Cool will do that.

Reply:
ping on this one